lib/ansible/modules/network/junos/junos_config.py

```diff
@@ -122,6 +122,12 @@
     type: bool
     default: 'no'
     version_added: "2.4"
+  check_commit:
+    description:
+      - This argument will check correctness of syntax; do not apply changes.
```

Comment:
A note can be added in the description to inform that this option can be used to confirm a confirmed commit operation.
Reference: https://www.juniper.net/documentation/en_US/junos/topics/topic-map/junos-configuration-commit.html
Check section: `Activating a Junos OS Configuration but Requiring Confirmation`